app/helpers/users_helper.rb

```diff
@@ -1,2 +1,88 @@
 module UsersHelper
+  def self.default_content
+    stories = []
```

Comment:
Is this default content there for all new users? Does it get removed once a user adds their own subscriptions?